components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -17,6 +17,8 @@ export interface StartWorkspaceModalProps {
     ide?: string;
     workspaceClass?: string;
     contextUrl?: string;
+    // If contextUrl is provided, setting `allowContextUrlChange` to true will allow it to be changed still
+    allowContextUrlChange?: boolean;
```

Comment:
I didn't want to change existing behavior for when the start workspace ui locks that context url, so adding this option felt like the safest approach.